src/lib/redis.ts

```
@@ -1,16 +1,39 @@
-export default redis;
\ No newline at end of file
+export default redis;
+
+if (process.env.NODE_ENV !== 'production') {globalThis.redisGlobal = redis;}
+
+export const getCached = async <T>(key: string): Promise<T | null> => {
+  const cachedData = await redis.get(key);
+  return cachedData ? JSON.parse(cachedData) : null;
+};
+
+export const setCached = async <T>(key: string, data: T, expirationInSeconds: number = 3600): Promise<void> => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)